### PR TITLE
Warmup cooldown program

### PR DESCRIPTION
**Pull request by OBackh**

### CODE REVIEW

File: pages/prep-and-follow-up.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                </StyledTableRow>
+              </thead>
+              <tbody>
+                {program.exerciseIds.map((exerciseId) => {
+                  const exercise = findWarmupCooldownExerciseById(exerciseId);
+
+                  if (!exercise) {
+                    return null;
+                  }
+                  const isDetailsVisible = showDetails[exercise.id] || false;
```
Comment (Inelukas) [inline pages/prep-and-follow-up.js:66]: i would recommend using another id here as using exercise.id leads to the problem that all exercise details in different warm-ups are toggled, which looks a bit buggy epecially in desktop view

File: pages/prep-and-follow-up.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                    <React.Fragment key={exercise.id}>
+                      <StyledTableRow key={exercise.id}>
+                        <StyledTableDataExercises>
+                          {exercise.name}
+                        </StyledTableDataExercises>
+                        <StyledTableData>{exercise.reps}</StyledTableData>
+                        <StyledTableData>{exercise.duration}</StyledTableData>
+                        <StyledTableData>
+                          <StyledInfoButton
+                            onClick={() => toggleDetails(exerciseId)}
```
Comment (Inelukas) [inline pages/prep-and-follow-up.js:77]: same as above

**Review by Inelukas — COMMENTED** [review]
The code looks good and everything's working! There's a small bug with the toggling of the details, maybe you should fix that before putting the story into QA

File: pages/prep-and-follow-up.js (hunk 1 above)
Comment (OBackh) [reply pages/prep-and-follow-up.js:66]: Thanks!
We just structered the data-lib new.

**Pull request merged**